app/controllers/users_controller.rb

```diff
@@ -8,4 +8,18 @@ def index
   def show
     @user = User.find(params[:id])
   end
+
+  def followings
+    @title = t('views.follow.show_follow.followings')
+    @user  = User.find(params[:id])
+    @users = @user.followings
+    render 'show_follow'
```

Comment:
```suggestion
    render 'friendships'
```

のようにするとpartial名がいい感じに抽象化できるかも？

Reply:
修正しました。([7b2c087](https://github.com/djkazunoko/fjord-books_app/pull/5/commits/7b2c08723c812bedeaf8193f57f9f399eebf5aa7))